### PR TITLE
NXP-28260: nuxeo-platform-3d marketplace - jsf plugins packaged in bundles-jsf-ui

### PR DESCRIPTION
**Pull request by ThierryCasanova**
…ndles-jsf-ui

### CODE REVIEW

File: marketplace/src/main/assemble/assembly.xml [hunk 1]
```diff
@@ -153,14 +212,14 @@
          basedir="${outdir}/marketplace" />
     <artifact:attach file="${outdir}/${maven.project.artifactId}-${maven.project.version}.zip" type="zip" />
 
-    <!-- cleanups -->
+    <!-- cleanups  -->
```
Comment (RSalem07) [inline marketplace/src/main/assemble/assembly.xml]: extra space

File: marketplace/src/main/assemble/assembly.xml [hunk 2]
```diff
@@ -153,14 +212,14 @@
...
     <delete dir="${outdir}/nuxeo-nxr-all" failonerror="false" />
     <delete file="${outdir}/nuxeo-nxr-server-${nuxeo.distribution.version}.zip" failonerror="false" />
     <delete file="${outdir}/nuxeo-nxr-jsf-ui-${nuxeo.distribution.version}.zip" failonerror="false" />
     <delete file="${outdir}/nuxeo-nxr-all-${nuxeo.distribution.version}.zip" failonerror="false" />
     <delete file="${outdir}/nxr-${maven.project.version}.zip" failonerror="false" />
-  </target>
 
-</project>
+  </target>
+</project>
```
Comment (RSalem07) [inline marketplace/src/main/assemble/assembly.xml]: add a new line at the file

**Review by RSalem07 — CHANGES_REQUESTED** [review]
i think it's ok, we will the review of thomas, i let some comments

File: pom.xml [hunk 3]
```diff
@@ -52,6 +52,12 @@
         <version>${nuxeo.dam.nxr.version}</version>
         <type>zip</type>
       </dependency>
+      <dependency>
+        <groupId>org.nuxeo.ecm.platform</groupId>
+        <artifactId>nuxeo-platform-3d-core-dependencies</artifactId>
+        <type>pom</type>
+        <version>${nuxeo.distribution.version}</version>
+      </dependency>
```
Comment (troger) [inline pom.xml]: Dependency management for `nuxeo-platform-3d-core-dependencies` should not be here but in the `addons/pom.xml` like other `nuxeo-platform-3d-*` dependencies.
https://github.com/nuxeo/nuxeo/blob/10.10/addons/pom.xml#L1056

**Review by troger — CHANGES_REQUESTED** [review]

**Review by RSalem07 — APPROVED** [review]

File: marketplace/pom.xml [hunk 4]
```diff
@@ -48,18 +43,10 @@
       <type>zip</type>
       <scope>provided</scope>
     </dependency>
-
```
Comment (NourNuxeo) [inline marketplace/pom.xml:51]: This is a clean up. We don't do that in HF.

File: marketplace/src/main/assemble/assembly.xml [hunk 5]
```diff
@@ -161,6 +220,6 @@
     <delete file="${outdir}/nuxeo-nxr-jsf-ui-${nuxeo.distribution.version}.zip" failonerror="false" />
     <delete file="${outdir}/nuxeo-nxr-all-${nuxeo.distribution.version}.zip" failonerror="false" />
     <delete file="${outdir}/nxr-${maven.project.version}.zip" failonerror="false" />
-  </target>
 
+  </target>
```
Comment (NourNuxeo) [inline marketplace/src/main/assemble/assembly.xml]: Just as clean up, we don't do formats in HF.

**Review by troger — APPROVED** [review]

**Pull request merged**